src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -85,9 +85,7 @@ export const NavBar: React.FC = track(
   }
 
   return (
-    <>
-      <NavBarSkipLink />
-
+    <Sticky zIndex={2}>
```

Comment:
For some reason I feel like this one line of code is gonna fix a bunch of future things

Reply:
Have we run into too many z-index wars in recent code? I suppose there may have been a reason this was `100`...

Reply:
I generally tend to think that if a z-index is greater than `1` something is wrong. Here I set it to `2` so that things scrolling past it that are `1` don't overlap. Though I wonder if I should really just portal this to the end of the page as if it were a modal so that it naturally stacks using `1`.

Reply:
This _was_ 100 because of old force app shell. For new pages 2 should work, but I'd check some old pages to see if things are ok.